build/openwisp_dashboard/Dockerfile

```diff
@@ -1,13 +1,18 @@
 FROM python:3.7-alpine AS BASE
 
-WORKDIR /opt/openwisp/
-RUN apk add --update --no-cache build-base libffi-dev openssl-dev postgresql-dev python3-dev zlib-dev jpeg-dev musl-dev gettext linux-headers git --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing gdal-dev geos-dev
-
 COPY ./openwisp_dashboard/requirements.txt /opt/openwisp/requirements.txt
-RUN pip install -r requirements.txt
+RUN apk add --update --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
```

Comment:
is it good to use `http://dl-cdn.alpinelinux.org/alpine/edge/testing`?

From https://wiki.alpinelinux.org/wiki/Edge

`Warning: "edge" is under constant development so be careful using it in production. It is possible that bugs in "edge" could cause data loss or could break your system.`

@2stacks @hispanico what do you think?

It seems to me that we should use something safer.

Reply:
Currently `gdal-dev geos-dev` are only available in `edge`, alternative would be to compile them, I was initially compiling them but that takes a lot of time so for convenience I moved to this. 
Let me know if compiling them sounds better. :-)